.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+  # - repo: https://github.com/astral-sh/ruff-pre-commit
+  # # Ruff version.
+  #   rev: v0.9.0
+  #   hooks:
+  #     # Run the linter.
+  #     - id: ruff
+  #       args: [ --fix ]
+  #     # Run the formatter.
+  #     - id: ruff-format
+  #- repo: https://github.com/pre-commit/mirrors-clang-format
+  #  rev: v15.0.7
+  #  hooks:
+  #    - id: clang-format
+  #      args: ["--sort-includes=false"]
```

Comment:
Nitpick: no reason to include this in this commit

This commit could be squashed with the following one